server/prisma/migrations/20240527134914_switch_to_uuids/migration.sql

```diff
@@ -0,0 +1,793 @@
+ALTER TABLE "TournamentGold" DROP COLUMN "id";
+ALTER TABLE "TournamentGold" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "TournamentGold" ADD CONSTRAINT "TournamentGold_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "TournamentXp" DROP CONSTRAINT IF exists "TournamentXp_pkey";
+ALTER TABLE "TournamentXp" ADD COLUMN "temp_id" INT;
+ALTER TABLE "TournamentXp" ADD COLUMN "temp_bruteId" INT;
+UPDATE "TournamentXp" SET "temp_id" = "id";
+UPDATE "TournamentXp" SET "temp_bruteId" = "bruteId";
+ALTER TABLE "TournamentXp" DROP COLUMN "id";
+ALTER TABLE "TournamentXp" DROP COLUMN "bruteId";
+ALTER TABLE "TournamentXp" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "TournamentXp" ADD COLUMN "bruteId" uuid  NOT NULL;
+ALTER TABLE "TournamentXp" ADD CONSTRAINT "TournamentXp_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "WorkerJob" DROP CONSTRAINT IF exists "WorkerJob_pkey";
+ALTER TABLE "WorkerJob" ADD COLUMN "temp_id" INT;
+UPDATE "WorkerJob" SET "temp_id" = "id";
+ALTER TABLE "WorkerJob" DROP COLUMN "id";
+ALTER TABLE "WorkerJob" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "WorkerJob" ADD CONSTRAINT "WorkerJob_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "_BruteReportToUser" ADD COLUMN "temp_A" INT;
+UPDATE "_BruteReportToUser" SET "temp_A" = "A";
+ALTER TABLE "_BruteReportToUser" DROP COLUMN "A";
+ALTER TABLE "_BruteReportToUser" ADD COLUMN "A" uuid NOT NULL ;
+
+-- AlterTable
+ALTER TABLE "_BruteToTitle" ADD COLUMN "temp_A" INT;
+UPDATE "_BruteToTitle" SET "temp_A" = "A";
+ALTER TABLE "_BruteToTitle" DROP COLUMN "A";
+ALTER TABLE "_BruteToTitle" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_BruteToTitle" ADD COLUMN "temp_B" INT;
+UPDATE "_BruteToTitle" SET "temp_B" = "B";
+ALTER TABLE "_BruteToTitle" DROP COLUMN "B";
+ALTER TABLE "_BruteToTitle" ADD COLUMN "B" uuid NOT NULL;
+
+-- AlterTable
+ALTER TABLE "_BruteToTournament" ADD COLUMN "temp_A" INT;
+UPDATE "_BruteToTournament" SET "temp_A" = "A";
+ALTER TABLE "_BruteToTournament" DROP COLUMN "A";
+ALTER TABLE "_BruteToTournament" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_BruteToTournament" ADD COLUMN "temp_B" INT;
+UPDATE "_BruteToTournament" SET "temp_B" = "B";
+ALTER TABLE "_BruteToTournament" DROP COLUMN "B";
+ALTER TABLE "_BruteToTournament" ADD COLUMN "B" uuid NOT NULL ;
+
+-- AlterTable
+ALTER TABLE "_Opponents" ADD COLUMN "temp_A" INT;
+UPDATE "_Opponents" SET "temp_A" = "A";
+ALTER TABLE "_Opponents" DROP COLUMN "A";
+ALTER TABLE "_Opponents" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_Opponents" ADD COLUMN "temp_B" INT;
+UPDATE "_Opponents" SET "temp_B" = "B";
+ALTER TABLE "_Opponents" DROP COLUMN "B";
+ALTER TABLE "_Opponents" ADD COLUMN "B" uuid NOT NULL ;
+
+
+-- UPDATE DATAS
+UPDATE "Achievement" SET "bruteId" = "Brute"."id" FROM "Brute" WHERE "Achievement"."temp_bruteId" = "Brute"."temp_id";
+UPDATE "Brute" SET "masterId" = "id" WHERE "temp_masterId" = "temp_id";
```

Comment:
I don't think this does what you want. This will affect 0 rows, since the condition is that the masterId equals the id, which cannot happen

Reply:
Yes it is, I want to put the new id of the brute who is the master so I'm checking in the temporary column for it ,  masterId's content is a uuid of a Brute

Reply:
Yes it can ^^ , because we have the previous id of a Brute in temp_id and I'm looking for his new uuid , temp_masterId has the old id of a brute so I have to compare with all the old ids of each brute to find the one that match and get his uuid. I assume that's how the masterId feild works